Parser.py

```diff
@@ -6,6 +6,7 @@
 import argparse
 import csv
 import logging
+import yaml
```

Comment:
YAML is a 3rd party library, currently we have no 3rd party dependencies. This was mostly intentional, as some systems my not have pip by default (such as a MacBook without a package manger). 

Are there any built in in libraries that can do similar things, can we package it in the repo somehow?